openapiart/bundler.py

```diff
@@ -833,22 +903,24 @@ def _generate_value_schema(
                     },
                 },
             }
+
             if "features" in xpattern and "count" in xpattern["features"]:
                 counter_schema["properties"]["count"] = {
                     "type": "integer",
+                    "format": "uint32",
```

Comment:
why is this hardcoded ? shouldn't it depend on bit length ?

Reply:
done